bridge/functions.php

```diff
@@ -115,6 +115,12 @@ function dd_wrap_autoloader()
             return $originalAutoloaderRegistered;
         }
 
+        // In case of composer, we need to be in front of it so our generated classes are loaded,
+        // including non-psr4 classes and files.
+        // This will not be required anymore once we move in the next release to a non wrapped autoloader.
+        dd_unregister_autoloader();
+        dd_register_autoloader();
```

Comment:
should this be after L124 ? `dd_untrace('spl_autoload_register');`
We do have reentrant guard in C tracer, but If that wouldn't work as expected for some reason. This will cause infinite recursion.